INSTALL.md

```diff
@@ -396,11 +396,11 @@ Launch XAMPP Control Panel and navigate to LibreEHR Setup in your browser: `\Lib
 Apache can be used on Mac to install LibreHealth. However, if you have installation problems, you can use MAMP.
 
 #### Setup
-Install MAMP (To prevent changing the PHP version, use version 3.5 and under).
+Install MAMP version 3.5.6 or under (Newer versions come with PHP 7.1 which is not compatible with LibreHealthEHR. If youdo install a newer version, make sure to use PHP versions under 7.1).
 
 #### Note: PHP must not be version 7.1 or above. Version 7.0 and 5.6 can both be used.
 
-Firstly, download [MAMP](https://mamp.info/en/). Currently, we do not support PHP versions 7.1 and above. Version 3.5.2 comes with PHP version 7.0. For ease, download this version as you would not have to go change the PHP version.
+Firstly, download [MAMP](https://mamp.info/en/) version 3.5.6. Currently, we do not support PHP versions 7.1. MAMP 3.5.6 comes with PHP 7.0 so you can use it without much configuration.
```

Comment:
Are you using numeric listing orrr please change Firstly to 1. to be uniform through out